molecule/default/tests/test_default.py

```diff
@@ -7,25 +7,18 @@
     os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('cinder_volume')
 
 
-def test_cinder_service(host):
+def test_cinder_service(skiptesthost):
```

Comment:
This results in an error for me because `skiptesthost` is not a testinfra magic fixture like `host` is.
```
E       fixture 'skiptesthost' not found
```

It looks like `pytest` supplies a skip facility that can be added just above the test `def` block to cause it to be unconditionally skipped. See docs: https://pytest.readthedocs.io/en/reorganize-docs/new-docs/user/skipping.html#unconditional-skip

Diff:
```
--- a/molecule/default/tests/test_default.py
+++ b/molecule/default/tests/test_default.py
@@ -7,6 +7,8 @@ testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
     os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('cinder_volume')


+import pytest
+@pytest.mark.skip(reason="this test needs to be executed on a host with the cinder cli available")
 def test_cinder_service(host):
     cmd = "sudo bash -c \"source /root/openrc; cinder service-list\""
     output = host.run(cmd)
```

I was able to successfully test this with the following results:
```
    ============================= test session starts ==============================
    platform linux2 -- Python 2.7.12, pytest-3.4.1, py-1.5.2, pluggy-0.6.0
    rootdir: /tmp/system_test_workingdir.8SKTAazt/rpc-openstack-system-tests/molecules/molecule-validate-cinder-deploy/molecule/default, inifile:
    plugins: testinfra-1.7.1
collected 3 items

    tests/test_default.py s..                                                [100%]

     generated xml file: /tmp/system_test_workingdir.8SKTAazt/rpc-openstack-system-tests/molecules/molecule-validate-cinder-deploy/molecule/default/test.xml
    ===================== 2 passed, 1 skipped in 6.02 seconds ======================
```